### PR TITLE
add max number of pixels per draw and configureable by admin

### PR DESCRIPTION
**Pull request by 0xaptosj**

### CODE REVIEW

File: move/sources/canvas/token.move [hunk 1]
```diff
@@ -602,6 +616,17 @@ module addr::canvas_token {
...
     }
 
+    public entry fun update_max_number_of_piexls_per_draw(
+        caller: &signer,
+        canvas: Object<Canvas>,
+        updated_max_number_of_pixels_per_draw: u64,
+    ) acquires Canvas {
+        let caller_addr = signer::address_of(caller);
+        assert_is_admin(canvas, caller_addr);
+        let canvas_ = borrow_global_mut<Canvas>(object::object_address(&canvas));
```
Comment (CapCap) [inline move/sources/canvas/token.move:626]: is this `let canvas_` a known convention? feels a bit weird

Comment (0xaptosj) [reply move/sources/canvas/token.move:626]: it is in this repo

**Review by CapCap — APPROVED** [review]
nice and simple

File: move/sources/canvas/token.move [hunk 2]
```diff
@@ -64,6 +61,12 @@ module addr::canvas_token {
     /// Vectors provided to draw were of different lengths.
     const E_INVALID_VECTOR_LENGTHS: u64 = 9;
 
+    /// The caller tried to call a function that requires collection owner privileges.
+    const E_CALLER_NOT_COLLECTION_OWNER: u64 = 10;
+
+    /// The caller exceeds the max number of pixels per draw.
+    const E_EXCEED_MAX_NUMBER_OF_PIXELS_PER_DRAW: u64 = 11;
+
```
Comment (CapCap) [inline move/sources/canvas/token.move:69]: can we make the error messages more user friendly? They show up in the txn result

Comment (0xaptosj) [reply move/sources/canvas/token.move:69]: can you give me a friendly version? sorry im terrible at writing

**Pull request merged**